cd_scripts/cd_functions.sh

```diff
@@ -136,13 +136,21 @@ write_perf_test_manifest(){
     echo "    PERF_TEST_END_NUMBER_OF_USERS: ${PERF_TEST_END_NUMBER_OF_USERS}" >> ${manifest}
     echo "    THRESHOLD_95TH_PERCENTILE_MILLIS: ${THRESHOLD_95TH_PERCENTILE_MILLIS}" >> ${manifest}
     echo "    THRESHOLD_MEAN_MILLIS: ${THRESHOLD_MEAN_MILLIS}" >> ${manifest}
+
+    echo "Performance test configuration:"
+    echo "PERF_TEST_START_NUMBER_OF_USERS: ${PERF_TEST_START_NUMBER_OF_USERS}"
```

Comment:
Does this need to be done in addition to above, can we not echo to console and the file?